### PR TITLE
[NFC] [AutoDiff] Gardening.

### PR DESCRIPTION
**Pull request by dan-zheng**
Miscellaneous organization/formatting/documentation.

### CODE REVIEW

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow macOS

File: lib/SILOptimizer/Mandatory/Differentiation.cpp [hunk 1]
```diff
@@ -4838,62 +4856,68 @@ class JVPEmitter final
   }
 
   /// Handle `tuple_extract` instruction.
-  ///   Original: y = tuple_element_addr x, <n>
-  ///   Tangent: tan[y] = tuple_element_addr tan[x], <n>
-  CLONE_AND_EMIT_TANGENT(TupleElementAddr, teai) {
+  ///   Original: y = tuple_extract x, <n>
+  ///    Tangent: tan[y] = tuple_extract tan[x], <n'>
+  ///                                           ^~~~
```
Comment (rxwei) [inline lib/SILOptimizer/Mandatory/Differentiation.cpp]: Align ^ with <

**Review by rxwei — APPROVED** [review]

File: lib/SILOptimizer/Mandatory/Differentiation.cpp (hunk 1 above)
Comment (dan-zheng) [reply lib/SILOptimizer/Mandatory/Differentiation.cpp]: Done in https://github.com/apple/swift/pull/27269/commits/c0b13d25b1b300e147ab12610b4a6d358d658ee1.

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Pull request merged**